lib/ably/models/protocol_message.rb

```diff
@@ -216,6 +216,11 @@ def has_transient_flag?
       flags & 16 == 16 # 2^4
     end
 
+    # @api private
+    def has_attach_resume_flag?
+      flags && 32 == 32 # 2^5
```

Comment:
This should be a single `&` (bitwise 'and' operator). At the moment it will return true whenever `flags` is truthy.

Reply:
Fixed. Thanks, good catch.